public/mixed_mich_2017/meta_study.txt

```diff
@@ -0,0 +1,7 @@
+type_of_cancer: mixed
+cancer_study_identifier: mixed_mich_2017
+name: Mixed Metastasis 500 (Michigan U 2017)
+short_name: MET500 (Michigan U 2017)
+description: Whole-exome and -transcriptome sequencing of 500 adult patients with metastatic solid tumor/primary normal pairs of diverse lineage and biopsy site.
```

Comment:
`-` infront of `transcriptome`

Reply:
it is quoted form the publication directly. I think the `-` in front just means another `whole`